CIPs/cip-granda_mento.md

```diff
@@ -0,0 +1,183 @@
+---
+cip: <to be assigned>
+title: Granda Mento
+author: <a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s), e.g. (use with the parentheses or triangular brackets): FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
+discussions-to: <URL>
+status: Draft
+type: <Standards Track, Meta or Informational>
+category (*only required for Standards Track): <Ring 0, 1, 2, 3>
+created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+requires (*optional): <CIP number(s)>
+replaces (*optional): <CIP number(s)>
+license: Apache 2.0
+---
+
+## Simple Summary
+
+Granda Mento is a mechanism to facilitate large CELO <-> stable token (e.g. cXXX) exchanges that aren't suitable via Mento or OTC. A new contract is created that has the authorization by Governance to exchange `CELO <-> stable token`. Exchanges via this contract must be approved by a multisig and can be vetoed by Governance.
```

Comment:
Albert mentioned Granda Mento would be for cUSD and cEUR for now, and not automatically for new stable coins. I think defining that here and using it consistently would be clarifying

Reply:
That's a great point. I was imagining that Granda Mento would be deployed for cUSD & cEUR now, but also for future stable tokens. @albertclabs does that align with what you were thinking?